packages/natds-themes/build/html/avon/light.html

```diff
@@ -13,7 +13,7 @@
 <body>
   <style>
     * {
-      font-family: 'Roboto';
+      font-family: 'Roboto, sans-serif';
```

Comment:
@arielwb idealmente, deveria ficar assim: `font-family: Roboto, sans-serif`, sem as aspas, senão o processador de CSS pode entender como um nome de fonte só. Se for estritamente necessário usar aspas, você poderia usar assim: `font-family: 'Roboto', 'sans-serif';`.

Reply:
listo ;)